### DISCUSSION TITLE
4:5 aspect ratio - broken images

### DISCUSSION THREAD

**Discussion by Dyzcord** [lllyasviel/Fooocus#2921] [Q&A]
greetings community, 
im trying to make some pictures for instagram with the popular 4:5 aspect ratio.
in almost every image im getting either on the right or bottom side broken/nonsense/blurry parts.

the affected right or bottom side changes with the resolution i choose.
i noticed that the results are forced to change in resolution. 
1000x1250 was forced to 1000x1248 and 1080x1350 to 1080x1344 and so on.
lower resolutions like 800x1000 and 992x1240 arent forced but getting bottom blurry parts.

ive tried to change the resolution with the developer debugger mode and with editing the config.

i dont use any special loras, settings etc. just images & prompts.

maybe someone out there got a solution/idea.

cheers :)

::speccs
13700k
3080@10gb
32gb ram
windows 10@22h2

some examples: 

![example_bottom1](https://github.com/lllyasviel/Fooocus/assets/162017542/8b0fc83f-cac6-4712-bd1b-28b6ebdbf91c)
![example_right2](https://github.com/lllyasviel/Fooocus/assets/162017542/f59213a0-e022-4dd3-a5f5-bc264808ba41)
![example_right1](https://github.com/lllyasviel/Fooocus/assets/162017542/a75f7196-71bb-4a08-9fbb-4dff3f5a9499)

**Comment by mashb1t** [comment]
Hey @Dyzcord, please check out the info in https://github.com/lllyasviel/Fooocus/pull/1617 and try to use a default near to 4:5 resolution SDXL has actually been trained on to see if this is beneficial or not.

**Comment by xhoxye** [comment]
Why not try these two ratios?
896×1152 ∣ 7:9
896×1088 ∣ 14:17
